pages/docs/guides/background-jobs.mdx

```diff
@@ -1,49 +1,51 @@
+export const description = `Define background jobs in just a few lines of code.`
+
 # Background jobs
-This guide will walk you through creating background jobs with retries in a few minutes.  Background tasks in Inngest are easy because:
+This guide will walk you through creating background jobs with retries in a few minutes.
+
+By running background tasks in Inngest:
 
-- You don't need to create queues, workers, or subscriptions
-- You can run background jobs on serverless functions without setting up infrastructure
+- You don't need to create queues, workers, or subscriptions.
+- You can run background jobs on serverless functions without setting up infrastructure.
 - You can enqueue jobs to run in the future, similar to a task queue, without any configuration.
 
 ## How to create background jobs
 
-There are two parts to background jobs:
+Background jobs in Inngest are executed in response to a trigger (an event or cron).
 
-1. Creating the function that runs in the background
-2. Triggering the function
+The example below shows a background job that uses an event (here called `user/created`) to send an email to new signups. It consists of two parts: creating the function that runs in the background and triggering the function.
 
-Let's get started on a background job that sends an email to new signups. This code creates
-a new Inngest function [as outlined in the function reference](/docs/reference/functions/create):
+<aside>
+👉 To be able to use the code below, remember to make first [serve your functions in your Inngest API](/docs/sdk/serve) to make it available to Inngest.
+</aside>
 
+### 1. Create a function that runs in the background
+
+<Row>
+  <Col>
+Let's walk through the code step by step:
+1. We [create a new Inngest function](/docs/reference/functions/create), which will run in the background any time the `user/created` event is sent to Inngest.
+2. We send an email reliably using the [`step.run()`](/docs/reference/functions/step-run) method. Every [Inngest step](/docs/steps) is automatically retried upon failure.
+3. We pause the execution of the function until a specific date using [`step.sleepUntil()`](/docs/reference/functions/step-sleep-until). The function will be resumed automatically, across server restarts or serverless functions. You don't have to worry about scale, memory leaks, connections, or restarts.
+4. We resume execution and perform other tasks.
+  </Col>
+<Col>
 <CodeGroup>
 ```ts {{ title: "v3" }}
 import { Inngest } from "inngest";
-
 const inngest = new Inngest({ id: "signup-flow" });
 
-export const fn = inngest.createFunction(
+export const sendSignUpEmail = inngest.createFunction(
   { id: "send-signup-email" },
-  { event: "user/created" }, // Trigger
+  { event: "user/created" },
```

Comment:
**suggestion (non-blocking)**: This legacy guide uses an old pattern for event naming. We generally recommend `<optional-prefix>/<object>.<action>` so in this case I would recommend `user.created`. It doesn't need to be part of this PR, but since we're updating this guide, it might be nice to make it look more in line with our other guides and recommended naming patterns.

Reply:
Thanks, I've corrected it in this guide and will be on a lookout in other portions of the docs.